Directory.Build.props

```diff
@@ -22,7 +22,7 @@
 
     <!-- Disable implicit package target fallback, and disable warning for when we explicitly add it (currently needed for
          Microsoft.ApplicationInsights) -->
-    <NoWarn>$(NoWarn);NU1701</NoWarn>
+    <NoWarn>$(NoWarn);NU1701;NU1507</NoWarn>
```

Comment:
💡 Comment needs to clarify explicitly what NU1701 and NU1507 are. Previously it was implied because only one warning was given.